flink-connector-kafka/src/test/java/org/apache/flink/connector/kafka/sink/KafkaSerializerWrapperTest.java

```diff
@@ -0,0 +1,28 @@
+package org.apache.flink.connector.kafka.sink;
+
+import org.apache.flink.streaming.connectors.kafka.testutils.SerializationTestBase;
+import org.apache.kafka.common.serialization.StringSerializer;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
Per the code style and quality guide, please don't use Mockito https://flink.apache.org/how-to-contribute/code-style-and-quality-common/#avoid-mockito---use-reusable-test-implementations

Reply:
Thanks for pointing it out. I have just updated it to get rid of Mockito.